### PR TITLE
EUI-2625-accessibility-policy:move accessibility policy common lib

### PR DESCRIPTION
**Pull request by rayliangatos**
…n component

Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
* Move accessiblity policy to common lib
* Will be used in MC/MO/AO

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
LGTMr

File: projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+</ul>
+
+<h2 class="govuk-heading-l" id="What-to-do-if-you-cannot-access-parts-of-this-website">Feedback and contact
+  information</h2>
+
+<p class="govuk-body">If you need information on this website in a different format like accessible PDF, large print,
+  easy read, audio recording or braille, you can:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>email: <a href="mailto:#"> HMCTSforms@justice.gov.uk</a></li>
```
Comment (paddybasi) [inline projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html:44]: shouldn't the href="mailto:" also contain the email address?

File: projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  <li>call: +44 (0) 300 123 1711</li>
```
Comment (paddybasi) [inline projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html:45]: just also wondering if this should have a <a href="tel:"> ...

File: projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+<p class="govuk-body">We’ll consider your request and get back to you in 10 working days.</p>
+
+<h2 class="govuk-heading-l" id="Reporting-accessibility-problems-with-this-website">Reporting accessibility problems
+  with this website</h2>
+
+<p class="govuk-body">We’re always looking to improve the accessibility of this website. If you find any problems not
+  listed on this page or think we’re not meeting accessibility requirements, contact:</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>email: <a href="mailto:#"> customer.service@justice.gov.uk</a></li>
```
Comment (paddybasi) [inline projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html:58]: as above

**Review by paddybasi — APPROVED** [review]
Approved, but a few comments to note

File: projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html (hunk 1 above)
Comment (rayliangatos) [reply projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html:44]: thanks, this is copied from the prototype and I'm not aware it mailto a #, will change it now

File: projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html (hunk 2 above)
Comment (rayliangatos) [reply projects/exui-common-lib/src/lib/components/accessibility/accessibility.component.html:45]: The requirement doesn't need a a for call, so will leave it

**Pull request merged**